clients/admin-ui/src/features/connection-type/types.ts

```diff
@@ -24,16 +24,20 @@ export type ConnectionTypeParams = {
   system_type?: SystemType;
 };
 
+export type ConnectionTypeSecretSchemaProperty = {
```

Comment:
I had to add quite a bit of new fields to this schema so I created a new type to simplify usage.